src/api/api-alephium.ts

```diff
@@ -429,12 +429,14 @@ export interface CompileContractResult {
   fields: FieldsSig
   functions: FunctionSig[]
   events: EventSig[]
+  warnings: string[]
 }
 
 export interface CompileScriptResult {
   bytecodeTemplate: string
   fields: FieldsSig
   functions: FunctionSig[]
+  warnings: string[]
```

Comment:
Do we need to print warnings when compiling scripts and contracts?

Reply:
I forgot to handle warning. I will create another PR. I am not sure if we should print the warnings or we should save it to a file for now.